### ISSUE TITLE
Add range details in tooltip for Context size

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#42590]
### Description

Add range details in tooltip for Context size

### Steps to reproduce

1. Add a custom model
2. Enter a custom context value `< 4000` or > `1.23B`
3. Save model
4. Edit model again
5. Context size is reset to default `4000` value as expected 
6. Tooltip doesn't mention the allowed range

### Actual result

https://github.com/user-attachments/assets/f2ab6a83-2a96-49a0-8a31-134e55ceb860

### Expected result

Should save custom value

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.74.15 Chromium: 131.0.6778.85 (Official Build) beta (64-bit)
-- | --
Revision | d1293587128488865fa61e9fa277dd54aa2270b0
OS | Windows 11 Version 23H2 (Build 22631.4460)

### Channel information

- [ ] release (stable)
- [x] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc: @jonathansampson

**Comment by srirambv** [comment]
> Perhaps we need to improve the messaging when a value is not valid.

Agreed. If we can have a range specified then its much more useful. I did try with reasonable value and it worked. I'll update the issue description and steps

**Comment by mattmcalister** [comment]
works for me
